web/client/plugins/Search.jsx

```diff
@@ -166,44 +166,128 @@ const SearchResultList = connect(selector, {
  * a **wfs** service look like this:
  * ```
  * {
- *      "type": "wfs",
- *      "priority": 2,
- *      "displayName": "${properties.propToDisplay}",
- *      "subTitle": " (a subtitle for the results coming from this service [ can contain expressions like ${properties.propForSubtitle}])",
- *      "options": {
- *        "url": "/geoserver/wfs",
- *        "typeName": "workspace:layer",
- *        "queriableAttributes": ["attribute_to_query"],
- *        "sortBy": "ID",
- *        "srsName": "EPSG:4326",
- *        "maxFeatures": 4,
- *        "blacklist": [... an array of strings to exclude from the final search filter ]
- *      },
- *      "nestedPlaceholder": "Write other text to refine the search...",
- *      "nestedPlaceholderMsgId": "id contained in the localization files i.e. search.nestedplaceholder",
- *      "then": [ ... an array of services to use when one item of this service is selected],
- *      "geomService": { optional service to retrieve the geometry}
- *  }
+ *  "type": "wfs",
+ *  "priority": 3,
+ *  "displayName": "${properties.propToDisplay}",
+ *  "subTitle": " (a subtitle for the results coming from this service [ can contain expressions like ${properties.propForSubtitle}])",
+ *  "options": {
+ *    "url": "/geoserver/wfs",
+ *    "typeName": "workspace:layer",
+ *    "queriableAttributes": ["attribute_to_query"],
+ *    "sortBy": "id",
+ *    "srsName": "EPSG:4326",
+ *    "maxFeatures": 20,
+ *    "blacklist": [... an array of strings to exclude from the final search filter ]
+ * },
+ *  "nestedPlaceholder": "Write other text to refine the search...",
+ *  "nestedPlaceholderMsgId": "id contained in the localization files i.e. search.nestedplaceholder",
+ *  "then": [ ... an array of services to use when one item of this service is selected],
+ *  "geomService": { optional service to retrieve the geometry }
  *
  * ```
- * The typical nested service needs to have some additional parameters:
+ * a service, may have nested services, that allow you to search in several steps,
+ * </br> (e.g *search for a street and in the next step search for the street number.*)
+ * </br>The typical service with nested service needs to have some additional parameters, like **nestedPlaceholder**, **then**
+* @prop {string} cfg.searchOptions.services[].nestedPlaceholder the placeholder will be displayed in the input text, after you have performed the first search.
```

Comment:
> * **all the information** for configuring is present in JSDoc, and not sparse in two places. For instance I noticed that `nestedPlacehodler` is not present in JSDoc

HI @offtherailz about `nestedPlaceholder` is present in the jsdoc, is described like a prop :)